docs/development/releasing.rst

```diff
@@ -115,7 +115,12 @@ packages that use the full bugfix/maintenance branch approach.)
 
    Don't forget to remove any non-committed files with::
 
-      $  git clean -dfx
+      $ git clean -dfx
+
+#. Make sure the source distribution doesn't inherit limited permissions
+   following you default umask::
+
+     $ umask 0022
```

Comment:
That wasn't true for me, but we can revisit later.  Go ahead and merge when ready.